### PR TITLE
Firebase access

### PR DESCRIPTION
**Pull request by CrizR**

### CODE REVIEW

File: data/models/account.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ const (
 
 // Account represents details for a particular user account.
 type Account struct {
+	AccountType AccountType
 	ID         int
```
Comment (xbrett) [inline data/models/account.go]: ID should be first, it will be primary key for the account

File: data/dataAccess.go [hunk 2]
```diff
@@ -1,4 +1,11 @@
 package data
 
 type DataAccess interface {
-}
+	// Not entirely sure what to the return type is going to be here
+	CreateUser([]string) (result string, err error)
```
Comment (xbrett) [inline data/dataAccess.go]: Its not typical to specify the name of the return results in go although you can do it. I would recommend (string, err)

File: data/dbConnect.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+)
+
+type DB struct {
+	ecclesiaDb *firebase.App
+}
+
+//The fuck, why is this not working >
+opt, err := option.WithCredentialsFile("keys/ecclesia-firebase-key.json")
+
+app, err := firebase.NewApp(context.Background(), nil, opt)
```
Comment (xbrett) [inline data/dbConnect.go]: this should all be in func New()
then it should work

File: main.go [hunk 4]
```diff
@@ -1,9 +1,74 @@
 package main
```
Comment (xbrett) [inline main.go:2]: this looks ok, but I am going to move it all to the right place later tonight

**Review by xbrett — CHANGES_REQUESTED** [review]
See comments in code

File: data/dataAccess.go (hunk 2 above)
Comment (CrizR) [reply data/dataAccess.go]: Sounds good

File: main.go (hunk 4 above)
Comment (CrizR) [reply main.go:2]: Ah alright

**Review by xbrett — APPROVED** [review]

**Pull request merged**